tests/test_external.py

```diff
@@ -62,6 +62,66 @@ def test_get_via_https_key_cert():
         pass
 
 
+def test_get_via_https_key_cert_password():
+    #  At this point I can only test
+    #  that the key and cert files are passed in
+    #  correctly to httplib. It would be nice to have
+    #  a real https endpoint to test against.
+    http = httplib2.Http(timeout=2)
+    http.add_certificate("akeyfile", "acertfile", "bitworking.org", "apassword")
+    try:
+        http.request("https://bitworking.org", "GET")
```

Comment:
Still trying to hit external website.

Reply:
> Test without relying on external network.
The test without relying on external network is implemented - test_get_via_https_key_cert_password_with_pem_local_server.

> Still trying to hit external website.
Just to clarify, do you expect all tests to use local server? There are quite a few tests which do that in test_external.py file. Or are you referring to newly added tests only?

Reply:
They are all transient, were created because local server stub was not implemented or not capable of required features at that time. See file docstring:

```
"""These tests rely on replies from public internet services

TODO: reimplement with local stubs
"""
```